### PR TITLE
Wallet integration

### PR DESCRIPTION
**Pull request by hariria**
## Motivation

https://user-images.githubusercontent.com/31301117/165797937-5ec41b07-6b24-48aa-8013-e0dfc191c827.mov

Addressing issues here: https://github.com/aptos-labs/aptos-core/issues/598

- [x] View the current test_coin balance
- [x] Fund the wallet through the faucet
- [x] Send coins to given address

Also did the following:
- [x] Redesign login page
- [x] Account state persists to localStorage
- [x] Error behaviors for invalid accounts
- [x] Loading states for faucet / submit transfer
- [x] Higher order component + dark mode 👿 🌚 

This PR was getting large... to do in upcoming PRs

- [ ] Invalid address for transfer behavior
- [ ] Invalid amount for transfer behavior (ie. what if they send -1 tokens?)
- [ ] Custom amounts for faucet?
- [ ] Create new wallet 
- [ ] Help page
- [ ] User settings / public key / private key / mnemonic?
- [ ] Sign out / wipe local storage
- [ ] Update linter

### Have you read the [Contributing Guidelines on pull requests](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

(You must have submitted a [signed CLA](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#contributor-license-agreement) that includes your GitHub handle prior to us accepting and landing your work. Write your answer here.)

## Test Plan

See video above

### CODE REVIEW

File: ecosystem/web-wallet/src/pages/Wallet.tsx [hunk 1]
```diff
@@ -1,20 +1,224 @@
...
+export const getAccountResources = async ({
+  nodeUrl = devnetNodeUrl,
+  address
+}: GetAccountResourcesProps) => {
+  const client = new AptosClient(nodeUrl)
+  if (address) {
+    const accountResources = await client.accounts.getAccountResources(
+      address
+      // {},
+      // { withCredentials: true, headers: { 'Access-Control-Allow-Credentials': true, credentials: true } }
```
Comment (kent-white) [inline ecosystem/web-wallet/src/pages/Wallet.tsx:57]: remove?

File: ecosystem/web-wallet/src/pages/Wallet.tsx [hunk 2]
```diff
@@ -1,20 +1,224 @@
...
+export const PopoverTrigger: React.FC<{ children: React.ReactNode }> =
+  OrigPopoverTrigger
+
+interface GetAccountResourcesProps {
+  nodeUrl?: string;
+  address?: string;
+}
+
+const devnetNodeUrl = 'https://fullnode.devnet.aptoslabs.com'
+const faucetUrl = 'https://faucet.devnet.aptoslabs.com'
```
Comment (kent-white) [inline ecosystem/web-wallet/src/pages/Wallet.tsx:46]: move these to `constants.ts`

File: ecosystem/web-wallet/src/pages/Wallet.tsx [hunk 3]
```diff
@@ -1,20 +1,224 @@
...
+          >
+            <PopoverTrigger>
+              <Button
+                isLoading={isTransferLoading}
+                isDisabled={isTransferLoading}
+                leftIcon={<IoIosSend />}
+              >
+                Send
+              </Button>
+            </PopoverTrigger>
```
Comment (kent-white) [inline ecosystem/web-wallet/src/pages/Wallet.tsx:186]: We should probably break some of these out into their own components, but not a big deal for now

**Review by kent-white — APPROVED** [review]
SICKKKKKKKKK

File: ecosystem/web-wallet/src/pages/Wallet.tsx (hunk 1 above)
Comment (hariria) [reply ecosystem/web-wallet/src/pages/Wallet.tsx:57]: oops

**Comment by kent-white** [comment]
you should cleanup your commits, I don't think those messages are what you wanted.

**Pull request merged**